neuralprophet/time_net.py

```diff
@@ -9,7 +10,17 @@
 import torch.nn as nn
 import torchmetrics
 
-from neuralprophet import configure, utils
+from neuralprophet import configure, utils_torch
+from neuralprophet.utils import (
+    check_for_regularization,
+    config_events_to_model_dims,
+    config_regressors_to_model_dims,
+    config_seasonality_to_model_dims,
+    reg_func_events,
+    reg_func_regressors,
+    reg_func_season,
+    reg_func_trend,
+)
```

Comment:
[for reviewer] importing utils directly caused an cyclic dependency error, thus we import all functions seperatly